lib/Swagger/src/Discussion/Models/UserDataResponse.php

```diff
@@ -0,0 +1,263 @@
+<?php
+/**
+ * UserDataResponse
+ *
+ * PHP version 5
+ *
+ * @category Class
+ * @package  Swagger\Client
+ * @author   http://github.com/swagger-api/swagger-codegen
+ * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache Licene v2
+ * @link     https://github.com/swagger-api/swagger-codegen
+ */
+
+/**
+ * discussion
+ *
+ * No descripton provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * OpenAPI spec version: 0.1-SNAPSHOT
+ * 
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen
+ * Do not edit the class manually.
+ */
+
+namespace Swagger\Client\Discussion\Models;
+
+use \ArrayAccess;
+
+/**
+ * UserDataResponse Class Doc Comment
+ *
+ * @category    Class */
+/** 
+ * @package     Swagger\Client
+ * @author      http://github.com/swagger-api/swagger-codegen
+ * @license     http://www.apache.org/licenses/LICENSE-2.0 Apache Licene v2
+ * @link        https://github.com/swagger-api/swagger-codegen
+ */
+class UserDataResponse implements ArrayAccess
+{
+    /**
+      * The original name of the model.
+      * @var string
+      */
+    protected static $swaggerModelName = 'UserDataResponse';
+
+    /**
+      * Array of property to type mappings. Used for (de)serialization
+      * @var string[]
+      */
+    protected static $swaggerTypes = array(
+        'has_reported' => 'bool',
+        'has_upvoted' => 'bool'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "has_upvoted" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)